.travis.yml

```diff
@@ -20,32 +20,6 @@ matrix:
       dist: trusty
       env: LARAVEL='5.4.*' XDEBUG=1
       group: edge
-    - php: 5.5.9
```

Comment:
This package allows installations from **PHP version 5.5.9**, so it is important to run the tests on this version.

See: https://github.com/tymondesigns/jwt-auth/blob/6e9457185ef4e4ff52eb54a73a64be58aa46847b/composer.json#L26

Reply:
I guess my suggestion is that it **shouldn't** allow those installations, on the basis that, that version of PHP hasn't been supported for many years.

Reply:
Okay, but this requires updating the **PHP constraint** in the `composer.json` file.

Remove **^5.5.9** from:
https://github.com/tymondesigns/jwt-auth/blob/6e9457185ef4e4ff52eb54a73a64be58aa46847b/composer.json#L26